### PR TITLE
Orca: Add requirements

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description

Add requirement_orca.txt for Orca for environment installation.

### CODE REVIEW

File: python/orca/src/setup.py [hunk 1]
```diff
@@ -91,7 +91,9 @@ def get_bigdl_packages():
             print("including", package)
     return bigdl_packages
 
-
+# The requirement list contains the main dependencies for users of orca. This list
+# should be carefully curated. If you change it, please reflect
+# the change in the matching file of requirements/orca.
```
Comment (hkvision) [inline python/orca/src/setup.py:96]: python/requirements/orca

File: python/requirements/orca/requirement_extra.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+boto3
+cloudpickle
+fsspec
+onnx
+onnxruntime
+Pillow
+py4j
+pyarrow
+tensorflow
+tensorflow_estimator
```
Comment (hkvision) [inline python/requirements/orca/requirement_extra.txt:11]: will tensorflow_estimator be installed as a dependency of tensorflow?

Comment (lalalapotter) [reply python/requirements/orca/requirement_extra.txt:11]: I think so, do we need to remove all the dependencies of tensorflow and pytorch?

File: python/orca/src/setup.py (hunk 1 above)
Comment (lalalapotter) [reply python/orca/src/setup.py]: fixed

File: python/requirements/orca/requirement_base.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# These are mirrored in python/orca/src/setup.py and python/dllib/src/setup.py 
+# as install_requires, which is what the users of the orca package will install.
+#
+# In short, if you change it here, PLEASE also change it in setup.py.
```
Comment (lalalapotter) [inline python/requirements/orca/requirement_base.txt:4]: shall we add this comment in each requirement?

Comment (hkvision) [reply python/requirements/orca/requirement_base.txt:4]: Maybe in base/automl and ray?

File: python/requirements/orca/requirement_extra.txt (hunk 2 above)
Comment (hkvision) [reply python/requirements/orca/requirement_extra.txt:11]: In this file, any other dependencies of tensorflow and pytorch?

File: python/requirements/orca/requirement_base.txt (hunk 3 above)
Comment (lalalapotter) [reply python/requirements/orca/requirement_base.txt:4]: Sure.

File: python/requirements/orca/requirement_extra.txt (hunk 2 above)
Comment (lalalapotter) [reply python/requirements/orca/requirement_extra.txt:11]: No, but in requirement_base.txt， `packaging`, `numpy`, `six` are also in the requirements of tensorflow.

Comment (hkvision) [reply python/requirements/orca/requirement_extra.txt:11]: Then it doesn't matter. Even if we don't install tensorflow, we have to install numpy and six.

**Review by hkvision — APPROVED** [review]

**Pull request merged**